My Amigo - html/meta/templates/speech-recognition-to-gif.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="utf-8">
+<meta content="width=device-width, initial-scale=1.0" name="viewport">
+<meta content="ie=edge" http-equiv="X-UA-Compatible">
+<title>Speak a Topic - View a Gif</title>
+<meta name="description" content="Demonstration of SpeechRecognition API that responds to voice commands and displays random animated gifs.">
+<!-- minimum viable  social tags -->
+<meta property="og:title" content="Speak a Topic - View a Gif"/>
+<meta property="og:description" content="Prototypes that demonstrate voice user interfaces for libraries."/>
+<meta property="og:image" content="http://www.lib.montana.edu/~jason/meta/img/clark-share-default.png"/>
+<meta property="og:url" content="http://www.lib.montana.edu/~jason/files/voice-user-interface/"/>
+<meta property="og:type" content="website"/>
+<meta name="twitter:creator" property="og:site_name" content="@jaclark"/>
```

Comment:
Very sad to see this, 
kindly close this PR, we do not tolerate plagiarism.

Reply:
if i remove these meta tags then it will be ok

Reply:
if not then i will close this pr